cilium/cmd/preflight_k8s_valid_cnp.go

```diff
@@ -69,28 +71,38 @@ func validateCNPs() error {
 		log.WithError(err).Fatal("Unable to connect to Kubernetes apiserver")
 	}
 
-	ctx, initCancel := context.WithTimeout(context.Background(), validateK8sPoliciesTimeout)
-	defer initCancel()
-	cnpErr := validateNPResources(ctx, "ciliumnetworkpolicies", "CiliumNetworkPolicy")
+	restConfig, err := k8s.CreateConfig()
+	if err != nil {
+		return fmt.Errorf("Unable to create rest configuration for k8s CRD: %w", err)
+	}
+	apiExtensionsClient, err := apiextensionsclient.NewForConfig(restConfig)
+	if err != nil {
+		return fmt.Errorf("Unable to create API extensions clientset for k8s CRD: %w", err)
```

Comment:
Travis reports:

```
cilium/cmd/preflight_k8s_valid_cnp.go:76:10: Errorf format %w has unknown verb w

cilium/cmd/preflight_k8s_valid_cnp.go:80:10: Errorf format %w has unknown verb w
```

This seems to be due to `%w` being a Go 1.13 feature. I think we just need to do:

```suggestion
		return fmt.Errorf("Unable to create rest configuration for k8s CRD: %s", err)
	}
	apiExtensionsClient, err := apiextensionsclient.NewForConfig(restConfig)
	if err != nil {
		return fmt.Errorf("Unable to create API extensions clientset for k8s CRD: %s", err)
```